packages/@unimodules/react-native-adapter/android/src/main/java/org/unimodules/adapters/react/ApplicationLifecycleDispatcher.kt

```diff
@@ -0,0 +1,24 @@
+package org.unimodules.adapters.react
+
+import android.app.Application
+import android.content.res.Configuration
+
+class ApplicationLifecycleDispatcher private constructor() {
+  companion object {
+
+    @JvmStatic
+    fun onApplicationCreate(application: Application) {
+      ExpoModulesPackageList.getPackageList()
+        .flatMap { it.createApplicationLifecycleListeners(application) }
+        .forEach { it.onCreate(application) }
```

Comment:
For this method you should be able to use a method reference:
```kt
ExpoModulesPackageList.getPackageList()
        .flatMap(Package::createApplicationLifecycleListeners)
        .forEach(ApplicationLifecycleListeners::onCreate)
```
But I don't have a strong option here. So you can leave it as it is.

Reply:
looks like not feasible b/c additional `application` parameter for createApplicationLifecycleListeners and onCreate.
have a playground for this: https://pl.kotl.in/yDauMooMP
does kotlin support partial parameter bound call like javascript bind or c++ std::bind?

Reply:
It won't work in that case. Sorry, my bad.

Reply:
not at all, thank you folks to teach me kotlin best practice from review 🥰